data-processing/tests/common/commands/test_fetch_s2_data.py

```diff
@@ -46,7 +50,7 @@ def test_makes_request_over_partner_api_when_token_present():
             command = FetchS2Metadata(Args(arxiv_ids=['fakeid']))
             command._mk_api_request("fakeid")
             mock_requests.get.assert_called_with(
-                "https://partner.semanticscholar.org/v1/paper/arXiv:fakeid",
+                f"https://api.semanticscholar.org/graph/v1/paper/arXiv:fakeid?{expected_fields_query}",
                 headers={"x-api-key": "some_token"}
             )
```

Comment:
I'd make the same changes noted above here too, by using the `requests` lib for constructing query string arguments.

Also I'd just copy the line for producing the list of reference related fields and inline it with each invocation here. Sure, you could use a single variable that's hosted to the top of the file -- but that's more indirection. Keeping things inline makes the code a bit easier to read, as there's no need to jump around.